### PR TITLE
Generate Discord Link

### PR DESCRIPTION
**Pull request by Pawoverlord**
This is to modify the HUB link from "https//\" to the link that we use to get to our discord from the wiki link. This is to hopefully avoid any confusion and allow players to easily locate our Discord. 

Kudos to plant from Discord finding this and bringing it to our attention! Thank you!

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
Does this actually compile?

**Comment by Pawoverlord** [comment]
I. . . Did not check. I'll do that when I get back home. Was super exhausted when trying to remember to do this. 😂

File: code/game/world.dm [hunk 1]
```diff
@@ -313,9 +313,9 @@ var/world_topic_spam_protect_time = world.timeofday
 
 	s += "<b>[station_name()]</b>";
 	s += " ("
-	s += "<a href=\"http://\">" //Change this to wherever you want the hub to link to.
+	s += "<a href=\"https://discord.com/invite/aXrM3SDyEw">" //Change this to wherever you want the hub to link to.
```
Comment (du-top) [inline code/game/world.dm:316]: ...yeah, you're missing a backslash here to escape the quotation mark: 
`"<a href=\"https://discord.com/invite/aXrM3SDyEw\">"`

**Comment by cdb-is-not-good** [comment]
marking as draft till it's fixed.

File: code/game/world.dm (hunk 1 above)
Comment (Pawoverlord) [reply code/game/world.dm:316]: Oh! I was playing around with it on my own code and couldn't quite figure out what I did wrong. 

Thank you!! 💜

**Comment by Pawoverlord** [comment]
I'll see about working on it Friday or Saturday. I also may have forgotten this PR for a few days after trying to resolve it on my side.

**Comment by cdb-is-not-good** [comment]
👍

**Comment by Trilbyspaceclone** [comment]
Seems stale

File: code/game/world.dm (hunk 1 above)
Comment (Trilbyspaceclone) [inline code/game/world.dm:316]:
```suggestion
	s += "<a href=\"https://discord.com/invite/aXrM3SDyEw\">" //Change this to wherever you want the hub to link to.
```

**Pull request merged**